tests/test_non_host.py

```diff
@@ -40,10 +40,32 @@ def test_custom_interpreter(
         expected -= {"setuptools", "wheel"}
     assert found == expected, out
 
-    monkeypatch.setattr(sys, "argv", [*cmd, "--graph-output", "something"])
-    with pytest.raises(SystemExit) as context:
-        main()
-    out, err = capfd.readouterr()
-    assert context.value.code == 1
-    assert not out
-    assert err == "graphviz functionality is not supported when querying non-host python\n"
+
+def test_custom_interpreter_with_local_only(
+    tmp_path: Path,
+    monkeypatch: pytest.MonkeyPatch,
+    capfd: pytest.CaptureFixture[str],
+) -> None:
+    venv_path = str(tmp_path / "venv")
+
+    result = virtualenv.cli_run([venv_path, "--system-site-packages", "--activators", ""])
+
+    cmd = [sys.executable]
+    monkeypatch.chdir(tmp_path)
+    cmd += [f"--python={result.creator.exe}"]
+    cmd += ["--local-only"]
```

Comment:
Let's remove the chdir monkeypatch since it doesn't look like it affects the test in any way.

Let's also combine `cmd` elements into one line:

```suggestion
    cmd = [sys.executable, f"--python={result.creator.exe}", "--local-only"]
```

Also, it doesn't actually matter what's in the first element since we are just parsing these arguments. Instead of `sys.exectuable`, let's add just the empty string for clarity.

Reply:
done